releases.json

```diff
@@ -1073,7 +1073,7 @@
       "highway"
     ],
     "versions": [
-      "1.2.0-1"
```

Comment:
This list is purely additive, we never remove entries from it, with entries ordered from newest to oldest.

Reply:
Yep thanks! Sorry should mark this as draft - the latest commit is just checking to see if the issues with the first commit are resolved by upstream changes